### ISSUE TITLE
Accuracy of training

### ISSUE THREAD

**Issue by Chinese-xingyi** [bearpaw/pytorch-classification#17]
Hi, I am try to use your modified model on cifar100.
But my alexnet's accuracy just can be around 42%; and vgg19_bn just can be 62%around ,63.46% most when the 130 epoch

**Comment by Chinese-xingyi** [comment]
I think the learning-rate and the epochs when I need to modified is specific for different models and different datasets.
I have tried to do some test about vgg19_bn on cifar10 and cifar100 -----modified learning rate of optimizer at 70th and 130th epoch  or   around these epochs .
Any setting can be optimized? Thank you !

**Comment by bearpaw** [comment]
Hi @Chinese-xingyi ，Did you follow these commands for training your model? https://github.com/bearpaw/pytorch-classification/blob/master/TRAINING.md#cifar-100
